src/routes/api/profile/experience/academic-experience/update/+server.ts

```diff
@@ -0,0 +1,30 @@
+import { BACKEND_BASE_URL } from '$env/static/private'
+import { json } from '@sveltejs/kit'
+
+export interface AcademicExperienceEditionPayload {
+  workExpId: number
+  organizationName: string
+  jobTitle: string
+  description: string
+  address: string
+  entryDate: string
+  departureDate: string
+}
+
+export async function POST({ request, fetch }: { request: Request; fetch: typeof window.fetch }) {
```

Comment:
```suggestion
export async function PUT({ request, fetch }: { request: Request; fetch: typeof window.fetch }) {
```
Como actualiza algo existente, debiera ser un PUT en vez de un POST. Y sí, SvelteKit sí lo soporta https://learn.svelte.dev/tutorial/other-handlers